examples/python/pytorch/resnet152_DDP_training.py

```diff
@@ -0,0 +1,95 @@
+import os
+import tempfile
+import torch,torchvision
+import torch.distributed as dist
+import torch.nn as nn
+import torch.optim as optim
+import argparse
+import torch.multiprocessing as mp
+from resnet_torch import resnet152
+import torchvision.transforms as transforms
+import time
+
+
+def setup(rank, world_size):
+    os.environ['MASTER_ADDR'] = 'localhost'
```

Comment:
this won't work with multiple nodes?

Reply:
Yeah, I would need to change the ip to the one from the rank 0 one right?